### PR TITLE
28th March wiki update

### PR DESCRIPTION
**Pull request by beingbravado**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,11 @@ CI (Continuous Integration) integrates code changes into a common branch of a sh
 
 CD (Continuous Delivery) is the practice of delivering code changes to production frequently and reliably. An example would be using GitHub Actions to run tests via CI, prepare a docker image for the app once the tests pass, and then trigger the deployment via CD.
 
+### CDN
```
Comment (prathamesh-sonpatki) [inline README.md:59]: @beingbravado Can you remove this? This is not really talking about o11y?

File: README.md [hunk 2]
```diff
@@ -344,6 +403,18 @@ Check https://www.brendangregg.com/usemethod.html for more details.
 
 ## V
 
+### Virtualization
```
Comment (prathamesh-sonpatki) [inline README.md:406]: @beingbravado Can you remove this? Not related to o11y directly.

**Review by prathamesh-sonpatki — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (beingbravado) [reply README.md:59]: Done

File: README.md (hunk 2 above)
Comment (beingbravado) [reply README.md]: Done

**Review by prathamesh-sonpatki — APPROVED** [review]

**Pull request merged**